webpack.config.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
         test: /\.s?css$/,
         use: [
           { loader: MiniCssExtractPlugin.loader },
-          { loader: 'css-loader' }, 
+          { loader: 'css-loader' },
           { loader: 'sass-loader' },
```

Comment:
hmm that one too 🤔 why css if we have sass?

Reply:
No idea, have to check. I just removed some trailing whitespaces here. 🤷‍♂️

Reply:
It seems as if `css-loader` is always needed, when you use `sass-loader` too.

```
ERROR in ./src/assets/main.scss
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
HookWebpackError: Module parse failed: Unexpected token (1:1)
File was processed with these loaders:
 * ./node_modules/sass-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
> ﻿.cookie-banner-iso{font-size:16px;/*!
```



> Chain the sass-loader with the [css-loader](https://github.com/webpack-contrib/css-loader) and the [style-loader](https://github.com/webpack-contrib/style-loader) to immediately apply all styles to the DOM or the [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin) to extract it into a separate file.

Source: https://www.npmjs.com/package/sass-loader